object_spawner:
  ros__parameters:
    spawn_box: True
    num_box: 5
    spawn_target: True
    target_type: "eight"
    num_target: 5
    x_rand: 4
    y_rand: 4